package.json

```diff
@@ -10,8 +10,8 @@
     "build": "./scripts/build.sh",
     "lint": "eslint -c .eslintrc.js '**/*.js'",
     "test": "nyc npm run test:unit && npm run test:routes && npm run lint",
-    "test:unit": "BABEL_DISABLE_CACHE=1 NODE_ENV=TEST mocha --recursive --require babel-core/register --require babel-polyfill --compilers js:babel-register ./test/unit/setup.js ./test/unit/t_*.js ./test/unit/**/*",
-    "test:routes": "HTTP_PROXY= HTTPS_PROXY= mocha --recursive --require babel-core/register --require babel-polyfill --compilers js:babel-register ./test/routes/setup.js ./test/routes/**/*"
+    "test:unit": "BABEL_DISABLE_CACHE=1 NODE_ENV=TEST mocha --recursive --require babel-core/register --compilers js:babel-register ./test/unit/setup.js ./test/unit/t_*.js ./test/unit/**/*",
```

Comment:
Apparently not:

    > BABEL_DISABLE_CACHE=1 NODE_ENV=TEST mocha --recursive --compilers js:babel-register ./test/unit/setup.js ./test/unit/t_*.js ./test/unit/**/*
    
    /home/cjwatson/src/canonical/build.snapcraft.io/build.snapcraft.io/node_modules/babel-core/lib/transformation/file/options/option-manager.js:176
              throw new ReferenceError(messages.get("pluginUnknown", plugin, loc, i, dirname));
              ^
    
    ReferenceError: Unknown plugin "transform-runtime" specified in "/home/cjwatson/src/canonical/build.snapcraft.io/build.snapcraft.io/.babelrc" at 3, attempted to resolve relative to "/home/cjwatson/src/canonical/build.snapcraft.io/build.snapcraft.io"
        at /home/cjwatson/src/canonical/build.snapcraft.io/build.snapcraft.io/node_modules/babel-core/lib/transformation/file/options/option-manager.js:176:17